modules/tftp/server.rb

```diff
@@ -153,7 +153,12 @@ def self.fetch_boot_file dst, src
   def self.choose_protocol_and_fetch(src, destination)
     case URI(src).scheme
     when 'http', 'https', 'ftp'
-      ::Proxy::HttpDownload.new(src.to_s, destination.to_s).start
+      options = {
+        :read_timeout => Proxy::TFTP::Plugin.settings.tftp_read_timeout || 10,
```

Comment:
Default values are defined in HttpDownload already, no need to define them here too?

Reply:
hmm I thought just in case the "defaults" are different for tftp vs general this would be a good spot. I can remove em if you dont think they are necessary.

Reply:
Nah, I think one set of defaults is enough. Moreover, ```HttpDownload``` isn't used anywhere else, it really ought to live in the tftp module.